### PR TITLE
Replace CSI Operator References with CSM Operator

### PR DESCRIPTION
**Pull request by KerryKovacevic**
# Description
Replace CSI Operator references/links with CSM Operator.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/996|

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by PeresKereotubo — DISMISSED** [review]

**Review by xuluna — DISMISSED** [review]

File: content/docs/csidriver/features/powerstore.md [hunk 1]
```diff
@@ -760,7 +760,7 @@ If such a node is not available, the pods stay in Pending state. This means they
...
 
 The attribute `storageCapacity.enabled` in `my-powerstore-settings.yaml` can be used to enabled/disabled the feature during driver installation .
-To configure how often driver checks for changed capacity set `storageCapacity.pollInterval` attribute. In case of driver installed via operator, this interval can be configured in the sample files provided [here](https://github.com/dell/dell-csi-operator/tree/master/samples) by editing the `capacity-poll-interval` argument present in the `provisioner` sidecar.
+To configure how often driver checks for changed capacity set `storageCapacity.pollInterval` attribute. In case of driver installed via operator, this interval can be configured in the sample files provided [here](https://github.com/dell/csm-operator/tree/main/samples) by editing the `capacity-poll-interval` argument present in the `provisioner` sidecar.
 
 **Note:**
 >This feature requires kubernetes v1.24 and above and will be automatically disabled in lower version of kubernetes.
```
Comment (gallacher) [inline content/docs/csidriver/features/powerstore.md:766]: Not necessarily related to this PR but this note can be removed.  For CSM 1.9 we only support 1.26, 1.27, and 1.28

**Review by gallacher — APPROVED** [review]
LGTM

**Review by xuluna — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]
Looks good to me

**Comment by shanmydell** [comment]
@KerryKovacevic : Please fix the liniting errors

**Comment by KerryKovacevic** [comment]
> @KerryKovacevic : Please fix the liniting errors

Lint error is part of a separate Installation Wizard issue that is known and being looked into by others.

**Review by sharmilarama — APPROVED** [review]

**Pull request merged**